staging-service-contract/src/main/resources/com/github/cafdataprocessing/services/staging/contract/swagger.yaml

```
@@ -163,4 +196,47 @@ components:
           type: array
           items:
             type: string
-            description: The batch identifier
\ No newline at end of file
+            description: The batch identifier
+    BatchStatusResponse:
+      title: BatchStatusResponse
+      description: The Response object with status of the batch
+      type: object
+      properties:
+        batchID:

```

It is `batchId` in other places in the contract.

```suggestion
        batchId:
```

---

Done.